src/main/java/com/metamx/common/io/smoosh/FileSmoosher.java

```diff
@@ -252,7 +263,7 @@ private Outer getNewCurrOut() throws FileNotFoundException
     final int fileNum = outFiles.size();
     File outFile = makeChunkFile(baseDir, fileNum);
     outFiles.add(outFile);
-    return new Outer(fileNum, new BufferedOutputStream(new FileOutputStream(outFile)), maxChunkSize);
+    return new Outer(fileNum, new FileOutputStream(outFile).getChannel(), maxChunkSize);
```

Comment:
In the meantime I was experimenting with this class, try-with-resources couldn't be used right here, because the outer is returned, and TWR will close channel. So I think it should be like this:
 1. Pass FOS into Outer instead of Channel, and save it as a field in Outer.
 2. Extract, (optionally wrap with Buffered..Channel) and store Channel in a field in Outer in the Outer's constructor.
 3. In addition to channel, close FOS in Outer's close.